common/src/main/java/com/ververica/platform/Utils.java

```diff
@@ -13,8 +13,12 @@
 import java.time.format.SignStyle;
 import java.time.temporal.ChronoField;
 import java.util.Date;
+import java.util.regex.Pattern;
 
 public class Utils {
+  public static Pattern SOURCE_FILENAME_COMPONENT_PATTERN =
```

Comment:
Move this to `PatternUtils`?

Reply:
I wanted to but that's in a different subproject...I could move that to 'common' as well though

Reply:
I didn’t realize it was a different sub project, feel free to leave it

Reply:
I moved it around...going ahead with the merge